CairoMakie/src/screen.jl

```diff
@@ -85,13 +85,15 @@ to_cairo_antialias(aa::Int) = aa
 * `pt_per_unit = 0.75`
 * `antialias::Union{Symbol, Int} = :best`: antialias modus Cairo uses to draw. Applicable options: `[:best => Cairo.ANTIALIAS_BEST, :good => Cairo.ANTIALIAS_GOOD, :subpixel => Cairo.ANTIALIAS_SUBPIXEL, :none => Cairo.ANTIALIAS_NONE]`.
 * `visible::Bool`: if true, a browser/image viewer will open to display rendered output.
+* `pdf_version::String = ""`: the version of output PDFs. Applicable options are `"1.4"`, `"1.5"`, `"1.6"`, `"1.7"`, or the default, `""`, which leaves the PDF version unrestricted.
 """
 struct ScreenConfig
     px_per_unit::Float64
     pt_per_unit::Float64
     antialias::Symbol
     visible::Bool
     start_renderloop::Bool # Only used to satisfy the interface for record using `Screen(...; start_renderloop=false)` for GLMakie
+    pdf_version::String
```

Comment:
Instead of storing any string here, and then validating later, I would create a struct `PdfVersion` or so which allows conversion from `String` so that you validate at creation of the screen config. Internally, the `PdfVersion` would store the integer that you defined elsewhere.

Reply:
Okay, I think I implemented the changes as requested.